README.md

```diff
@@ -24,5 +24,10 @@ and execute it:
 
 ## Dependencies
 
-ElasticSearch provides the search functionality, and must be running at the $ELASTIC_SEARCH_ENDPOINT listed in the environment variables.
+```bash
```

Comment:
I don't get this python version part. Perhaps it would be worth adding a sentence that would give some context. We're also using Python 3 in property API. In order to avoid confusion it would be best to replace the 2.7.5 version from the output to 3.4.2, for example.